templates/agenda-for-all-hands-call.md

```diff
@@ -7,7 +7,7 @@
 
 
 ## Agenda
-1. Roll call by alphabetical order (moderator)
+1. Roll call - see [timezone-based roll call](../admin-guides/timezone-rollcall.md) guide - _ensure notetaker is present_
```

Comment:
Agreed. I just think that that should be pointed out before the roll call. It doesn't happen during the roll call: "Oh, and you're the notetaker", it happens before. "Is the notetaker here? Great. Ok, roll call." 

Make sense?